docs/pages/deployment/configuration.md

```diff
@@ -4,19 +4,19 @@
 
 ### `LIVE_COMPOSITOR_API_PORT`
 
-API port. Defaults to 8001.
+API port. Defaults to 8081.
```

Comment:
I wasn't sure if it should be 8001 or 8081, but it was 8081 in code, so I decided to change docs